components/service/firefox-accounts/src/test/java/mozilla/components/service/fxa/FxaDeviceConstellationTest.kt

```diff
@@ -232,6 +239,68 @@ class FxaDeviceConstellationTest {
         assertEquals(ConstellationState(currentDeviceExpired.into(), listOf(testDevice2.into())), constellation.state())
     }
 
+    @Test
+    @ExperimentalCoroutinesApi
+    fun `polling for commands with real account`() = runBlockingTest {
+        val testScope = GlobalScope
+
+        testScope.launch {
+            // Initialize things so the megazord is happy
+            WorkManagerTestInitHelper.initializeTestWorkManager(testContext)
+            Megazord.init()
+
+            // Create a test account
+            val testAccount = spy(NativeFirefoxAccount(
+                    handle = Long.MIN_VALUE,
+                    persistCallback = null
+            ))
+
+            // Create out device constellation object that accepts the account
+            val testConstellation = FxaDeviceConstellation(
+                    account = testAccount,
+                    scope = CoroutineScope(coroutinesTestRule.testDispatcher) + SupervisorJob()
+            )
+
+            testConstellation.pollForCommandsAsync().await()
+
+            verify(testAccount.pollDeviceCommands())
+        }
+    }
+
+    @Test
+    fun `test sending one tab`() = runBlockingTest {
+        // Create a scope to run this test inside
+        val testScope = GlobalScope
+
+        testScope.launch {
+            // Initialize things so the megazord is happy
+            WorkManagerTestInitHelper.initializeTestWorkManager(testContext)
+            Megazord.init()
```

Comment:
Consider moving this to this test class' `init` block, otherwise you're calling `init` for each test.